migrate_pim/migrate_to_saas.rst

```diff
@@ -0,0 +1,157 @@
+
+  * This can give you a chance to run tests before your production instance is migrated
+
+* There is no need to manually update your PIM — all product feature and security updates are done automatically
+
+* There will be some downtime as we migrate your Flexibility instance to a Serenity one
+
+  * We will work with you to minimize the business impact of this downtime
+
+Before you begin
+----------------
+
+Check that your project terms include Serenity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To avoid any delays in migrating your Akeneo PIM to our most advanced cloud offering, please contact your Customer Success Manager to ensure
+they are aware of your migration and that your project terms include Serenity. 
+
+If any changes need to be made, your CSM can work with you to amend the terms.
+
+Check that your PIM is on the latest patch
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+    Beginning on 1 June 2023, all upgrades to Serenity will require the most recent patch of Akeneo PIM ``v7``, which was released in March 2023.
+
+    Instructions for upgrading a major version can be found in our article :doc:`./upgrade_major_version`
+
+    If you are upgrading before June 2023, you must be on the latest patch of Akeneo PIM ``v6``.
+
+If you are not on the latest patch, you may encounter problems that can delay your migration to Serenity. To avoid any problems,
+compare your PIM version (*ex:* ``6.0.68``) with the latest version of Akeneo PIM.
+
+To check if you need to upgrade, you can scroll to the bottom of your PIM's UI — if the PIM can be upgraded, you will see a message like ``New patch available: 6.0.82``.
+
+Guides to update the PIM to latest patch can be found here: :doc:`./apply_patch/index`
+
+Backup your PIM
+~~~~~~~~~~~~~~~
+
+We *strongly* advise you to make a MySQL backup of your data before you begin the migration process.
+
+To do so, please follow the directions in our :doc:`/cloud_edition/flexibility_mode/docs/system_administration` documentation.
+
+Disable custom code
+~~~~~~~~~~~~~~~~~~~
+
+Serenity does not support custom code within the PIM. In Serenity, all custom features use API connections and apps from the `App Store <https://apps.akeneo.com>`_ 
+(including `custom apps <https://api.akeneo.com/apps/create-custom-app.html>`_).
+
+**If you have custom code bundles, please** `remove them <https://docs.akeneo.com/master/maintain_pim/first_aid_kit/index.html#step-10-did-you-customize-your-pim>`_ 
+**before you begin the migration process.**
+
+If you cannot find a suitable replacement for your custom code, please contact your Customer Success Manager — they may be able to suggest changes
+or offer solutions that are compatibile with Serenity.
+
+The migration process
+---------------------
+
+Open a Support ticket
+~~~~~~~~~~~~~~~~~~~~~
+
+To begin the process to migrate from an On-Premise or Flexibility instance to a Serenity instance, you will need to open a ticket with Akeneo Support. 
```

Comment:
We could add a link to the correct helpdesk page: https://akeneo.atlassian.net/servicedesk/customer/portal/8/group/8/create/198

Reply:
```suggestion
To begin the process to migrate from an On-Premise or Flexibility instance to a Serenity instance, you will need to `open a ticket with Akeneo Support. <https://akeneo.atlassian.net/servicedesk/customer/portal/8/group/8/create/198>_` 
```